curator-framework/src/main/java/org/apache/curator/framework/api/transaction/TransactionCreateBuilder.java

```diff
@@ -18,17 +18,17 @@
  */
 package org.apache.curator.framework.api.transaction;
 
-import org.apache.curator.framework.api.ACLCreateModePathAndBytesable;
-import org.apache.curator.framework.api.ACLPathAndBytesable;
-import org.apache.curator.framework.api.Compressible;
-import org.apache.curator.framework.api.CreateModable;
-import org.apache.curator.framework.api.PathAndBytesable;
-
-public interface TransactionCreateBuilder<T> extends
-    PathAndBytesable<T>,
-    CreateModable<ACLPathAndBytesable<T>>,
-    ACLPathAndBytesable<T>,
-    ACLCreateModePathAndBytesable<T>,
-    Compressible<ACLCreateModePathAndBytesable<T>>
+public interface TransactionCreateBuilder<T> extends TransactionCreateBuilder2<T>
```

Comment:
I'm sure there's a reason for this approach, but can you enlighten me as to why you don't just add the withTtl() method to the TransactionCreateBuilder and not introduce the TransactionCreateBuilder2?